### ISSUE TITLE
Implement Schedulded Start

### ISSUE THREAD

**Issue by Empor-co** [Hyundai-Kia-Connect/hyundai_kia_connect_api#18]
@cdnninja as discussed in [#12 (comment)](https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/issues/12#issuecomment-1013718031) it might be interesting to also schedule the start for e.g. the next day.

**Comment by fuatakgun** [comment]
this package just responds to function calls, nothing about future trackings. You need to set your own scheduling logic and call it accordingly. Or you can use a home automation system as Home Assistant and manage it over there.

**Comment by fuatakgun** [comment]
please take a look at the second part of my previous post @Empor-co :)

**Comment by Empor-co** [comment]
I was under the impression that the schedule could be set/send to the car and the car deals with the triggering directly.

**Comment by fuatakgun** [comment]
I do now know :) if that is the case, we need your help to make this happen, or we can try to migrate it from bluelinky, if it is implemented.

**Comment by cdnninja** [comment]
Can you do this in the native mobile app for your car?

**Comment by Empor-co** [comment]
I haven't seen it in bluelinky, but the app offers it under Remote -> Scheduled charging.
There you can model 2 departure times, which can be used for 
- Scheduled for charging and
- Scheduled climate control
The target temperature and defrost can be set in the Scheduled climate control screen.

**Comment by cdnninja** [comment]
Which region are you in?

**Comment by Empor-co** [comment]
Hyundai / EU, please find attached a screenshot of the original app screen.
![charging_resized](https://user-images.githubusercontent.com/32220981/149678114-8015c733-19db-4f74-9b5d-a7c572f054bb.jpg)

**Comment by fuatakgun** [comment]
Good to know, as a phev user, i had overlooked at this, apparently i also have this functionality.

I need to setup my local traffic sniffer to extract the information.

Let's keep this open

**Comment by xpentil** [comment]
Great work on this project, everything works smooth on my setup.
This request is the last thing I need to truly be able to make it perfect, to automatically start the charge based on price and SOC.
Thank you for your awesome work!

**Comment by cdnninja** [comment]
Using home assistant logic it can be done today.  Just not with the built in schedule tool.  

Someone needs to sniff the eu traffic for this for us to make this part happen.

**Comment by xpentil** [comment]
Yes of course but if the internet fails around schedule, I ain't going to work the next day.

I will have time to help out sniffing in a few weeks.

Thank you

**Comment by CosAdrPala** [comment]
On the controls:
![image](https://github.com/user-attachments/assets/7cce4bba-78ee-4c4a-af26-2b8d88db7656)
Can you please add the possibility of changing the Off-peak start and end:
![image](https://github.com/user-attachments/assets/2885e9a5-81ba-4f6c-bf85-99328e93785b)
I can use only the granny charger (house socket) on the car and I do not have a Off-peak for EVs. My current tariff has 3 periods of cheap electricity and I would want to dynamically change the Off-peak period from HA.

**Comment by cdnninja** [comment]
This exists under: schedule_charging_and_climate
